src/workflows-app/utils/submission-utils.ts

```diff
@@ -477,3 +500,66 @@ export const validateInputs = (
     const inputMessage = validateInput(input, dataTableAttributes);
     return { name: 'input_name' in input ? input.input_name : input.field_name, ...inputMessage };
   });
+
+export const getInputTableData = (
+  configuredInputDefinition: InputDefinition[],
+  searchFilter: string,
+  includeOptionalInputs: boolean,
+  inputTableSort: { field: string; direction: boolean | 'asc' | 'desc' }
+): InputTableData[] => {
+  const val = _.flow(
+    (rows: InputDefinition[]) =>
+      rows.map((row, index): InputTableData => {
+        const { workflow, call, variable } = parseMethodString(row.input_name);
+        return {
+          taskName: call || workflow || '',
+          variable: variable || '',
+          inputTypeStr: renderTypeText(row.input_type),
+          configurationIndex: index,
+          optional: isInputOptional(row.input_type),
+          ...row,
+        };
+      }),
```

Comment:
If you want to lean in to the FP / `flow` style, this could also be written as:
```suggestion
    Utils.toIndexPairs,
    _.map(([index, row]: [number, InputDefinition]): InputTableData => {
      const { workflow, call, variable } = parseMethodString(row.input_name);
      return {
        taskName: call || workflow || '',
        variable: variable || '',
        inputTypeStr: renderTypeText(row.input_type),
        configurationIndex: index,
        optional: isInputOptional(row.input_type),
        ...row,
      };
    }),
```